app/viewers/embed/viewer/document_viewer.rb

```diff
@@ -27,15 +27,11 @@ def fullscreen?
         true
       end
 
-      def all_documents_location_restricted?
-        document_resource_files.all?(&:location_restricted?)
-      end
-
-      # this indicates if the PDF is downloadable (though it could be stanford only)
-      # Stanford only and location restrictions are handled via a separate authorization flow,
-      # since it is possible for people to do something about the restriction
+      # This indicates if the first PDF is downloadable (though it could be stanford only)
+      # as well as location restricted.  Authorization for other documents will be
+      # checked as the user clicks on items in the content side bar.
       def available?
-        document_resource_files.first&.downloadable?
+        !document_resource_files.first&.no_download?
```

Comment:
I'm still not clear on why we need this at all.  Can't we just run this all through the IIIF auth?

Reply:
The PDF Component html will not show the viewer at all if the "viewer.available?" is false.  Instead, it will just show a "content not available" section.  This change ensures the content not available section is not displayed unless the first file is a "no download" option. If we want to change that logic, I'd suggest tackling that in a different issue because there is probably more to streamline here (or to work out).